db/migrate/20231222202435_comment_table.rb

```diff
@@ -0,0 +1,11 @@
+class CommentsTable < ActiveRecord::Migration[7.1]
```

Comment:
- [ ] **[OPTIONAL]** You should rename this file to `...comments_table` to match the class name.